kano_init_flow/audio_screen.py

```diff
@@ -184,11 +185,11 @@ def __init__(self, win):
 
         self.template = Template(
             img_path=os.path.join(MEDIA_DIR, "Audio_See_the_light.png"),
-            title="Can you see the light?",
-            description="If the power plugs are connected correctly, " \
-                        "you should see a blue light.",
-            button1_text="YES",
-            button2_text="NO"
+            title=_("Can you see the light?"),
+            description=_("If the power plugs are connected correctly, " \
```

Comment:
I think that the backslash is not needed when a line is broken within a parenthesis, but it shouldn't be an issue